client/shared/src/mem.rs

```diff
@@ -12,18 +12,56 @@ const fn MiB(n: usize) -> usize {
 
 const STACK_ALIGNMENT: usize = 1024 * 63;
 
+/// The size of the stack. Its start is at address 0
 pub const GRAPHIC_STACK_SIZE: usize = MiB(4) + STACK_ALIGNMENT;
+
+/// The address of the Allocator structures (size: 1MiB)
 pub const ALLOCATOR_AREA_START: usize = GRAPHIC_STACK_SIZE;
-pub const SHARED_BUFFER_AREA_START: usize = ALLOCATOR_AREA_START + MiB(1);
+
+/// The address memory communication area. (size: 1MiB)
+/// It contains data needed for communication between main thread and logic thread.
+pub const COMMUNICATION_MEMORY_START: usize = ALLOCATOR_AREA_START + MiB(1);
```

Comment:
Maybe synchronization would be a more appropriate name?

Reply:
Maybe it is, but I'm still not completely sure, what it's gonna contain. But for sure the `elapsed_ms` field.

Reply:
Yes, I think this would be appropriate, since this communication occurs only in one direction and  serves the single purpose of synchronization.